### PR TITLE
added capability for a OCKDailyPageViewController to select a certain…

### PR DESCRIPTION
**Pull request by tommysarni**
… date to display

### CODE REVIEW

File: CareKit/CareKit/Lists/Controller/OCKDailyPageViewController.swift [hunk 1]
```diff
@@ -110,6 +110,10 @@ UIPageViewControllerDataSource, UIPageViewControllerDelegate {
 
     // MARK: - Properties
 
+    open func selectDate(_ date: Date, animated: Bool) {
+        calendarWeekPageViewController.selectDate(date, animated: animated)
```
Comment (erik-apple) [inline CareKit/CareKit/Lists/Controller/OCKDailyPageViewController.swift]:
```suggestion
        let previousDate = selectedDate
        guard !Calendar.current.isDate(previousDate, inSameDayAs: date) else { return }
        calendarWeekPageViewController.selectDate(date, animated: true)
        weekCalendarPageViewController(calendarWeekPageViewController, didSelectDate: date, previousDate: previousDate)
```

Comment (erik-apple) [reply CareKit/CareKit/Lists/Controller/OCKDailyPageViewController.swift]: Then the `pressedToday(sender:)` method on line 155 can be changed to just

```swift
@objc
private func pressedToday(sender: UIBarButtonItem) {
    selectDate(Date(), animated: true)
}
```

Comment (tommysarni) [reply CareKit/CareKit/Lists/Controller/OCKDailyPageViewController.swift]: Sounds Good !

**Comment by erik-apple** [comment]
Congrats on your first contribution @tommysarni! 🎊
I hope there will be more where that came from!

**Pull request merged**